cognite/client/_api/files.py

```diff
@@ -620,12 +626,53 @@ def download(
                 >>> c = CogniteClient()
                 >>> c.files.download(directory="my_directory", id=[1,2,3], external_id=["abc", "def"])
         """
+
         if isinstance(directory, str):
             directory = Path(directory)
+        assert directory.is_dir(), f"{directory} is not a directory"
```

Comment:
Asserts are not meant for the end user, raise an exception instead (`IsADirectoryError`)

Reply:
Sorry this was a stupid suggestion, the problem is that it's not a directory 😂 This should of course be `NotADirectoryError ` :hide: @CogniteJohan